csrc/host_ir/lower.cpp

```diff
@@ -435,10 +435,9 @@ bool HostIrLower::canLower(Expr* expr, bool ignore_inner_resharding) {
     // stream-parallelized on axis 0.
     auto* a = linear->inA()->as<TensorView>();
     auto* b = linear->inB()->as<TensorView>();
-    auto* bias = linear->bias()->as<TensorView>();
     auto* out = linear->out()->as<TensorView>();
-    return !isSharded(b) && !(linear->has_bias() && isSharded(bias)) &&
-        !isSharded(out) &&
+    return !isSharded(b) &&
+        !(linear->has_bias() && isSharded(linear->bias())) && !isSharded(out) &&
```

Comment:
TODO: remove this. It is a bug that will surface in `insert_resharding` so move it to a separate PR.

Reply:
FYI, https://github.com/NVIDIA/Fuser/pull/4208/files